docs/DesignTokens/Taxonomy.stories.mdx

```diff
@@ -0,0 +1,138 @@
+  - `forms` for defining the border color for input controls (text input, select, checkbox, radio input…).
+  - `controls` for controls you interact with like form fields and navigation.
+  - `actions` for tageting both buttons and links.
+- Component
+  - `button`, `input`, `checkbox`, `tooltip`, `modal`…
+- Element — For targeting elements within a component, similar to [BEM](https://getbem.com/).
+  - `icon` within a `button` component.
+
+### Base
+
+- Category
+  - `color`, `font`, `border`, `space`, `size`, `shadow`, `elevation`, `opacity`, `breakpoint`, `touch`, `time`
+- Property
+  - `background`, `text`, `border`, and `icon` for `color` category
+  - `family`, `size`, `weight`, `line-height`, and `letter-spacing` for `font` category
+  - `width`, `style`, `radius` for `border` category  
+- Concept and sub-concepts
+  - `feedback`, `action`, and `surface` for `color` category
+  - `display`, `heading` and `body` for `font` category 
+
+### Modifier
+
+- Variant — Distinguishes alternative use cases.
+  - Intent
+    - `info`, `success`, `warning`, and `cirtical` for `feedback` concept
+    - `conversion`, `positive`, `negative`, and `neutral` for `action`
+  - Emphasis — Prominence
+    - `primary`, `secondary`, and `tertiary` for `action` concept 
+- State — Properties based on interactive states.
+  - `default`, `hovered`, `active`/`pressed`, `focused`, `selected`, `visited`, `disabled`,  `error`
+- Scale — Choices within a predefined set of values, like varying size, space, shade, and other options.
+  - Ordinal: t-shirt sizes (xxs, xs, sm, md, lg, xl, xxl…).
+  - Sequential/stepped/numerical: base 100 (100, 200, 300…), base 1/index (0_5, 1, 2, 4…)
+  - Proportional: -1x, 0_5x, 1x, 2x…
+  - Keyword/Named:
+    - circle
+    - regular, tight, loose
+    - regular, medium, semi-bold, bold
+    - sans, serif, mono
+    - minimum
+    - ease-in, ease-out
+- Mode — Distinguish values across two or more surface/background settings on which elements appear. Use only when there’s a distinction (e.g., color and shadow for dark theme).
+  - `on-light`, `on-dark`, `on-light-hc`, `on-dark-hc`, or any colored surface like `invert`, `on-accent`, `on-primary`, `on-secondary`, `on-surface`.
+
+### Other naming rules
+
+- For compound words, use hyphens to separate the words: `line-height`, `on-primary`, `on-dark-hc`
+- For decimal values, replace the decimal point with an underscore: `0_5` for 0.5.
+- For negative values, use a hyphen: `-1`.
+- “Avoid dogmatically including all levels possible or duplicating token-tuples redundantly. Instead, include only the levels needed to sufficiently describe and distinguish purposeful intent.”
+
+Refer to [Naming Tokens in Design Systems](https://medium.com/eightshapes-llc/naming-tokens-in-design-systems-9e86c7444676) by Nathan Curtis for further examples.
+
+Exceptions to the rules described in Nathans's article:
+
+- Property comes before Concept and Sub-concept.
```

Comment:
Not sure I understand this rule nor how to apply it? Could we elaborate a bit more and add some context?

Reply:
Rewrote it for clarity.